XenoHolographic-it/IAR-SDK-Sample-iOS
# IAR-5059 Git hooks to prevent leaking of orgKeys

# Overview

Detects accidental inclusion of orgKey-like strings to the public repo

# Resources

Jira https://iversoft.atlassian.net/browse/IAR-5059

# Testing Instructions

Try to add a valid orgKey into a commit - it should cancel the commit with a message.

eg: "pk_org_DTyWqiM89nqPWHPbwyc2uMUgkr3vJ29A" (not valid, same size)

# Screenshots

![Screen Shot 2022-01-13 at 2 36 17 PM](https://user-images.githubusercontent.com/9361619/149407384-cd19ca90-03a3-4b0b-8fb8-91c02811eff9.png)

<img width="807" alt="Screen Shot 2022-01-13 at 2 53 05 PM" src="https://user-images.githubusercontent.com/9361619/149407524-8f85a8e8-3b5f-4e3c-b4e3-b92db3349d2e.png">
